submissions/Mifaresss/HTML-CSS-Popup/index.html

```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta charset="UTF-8">
+	<meta http-equiv="X-UA-Compatible" content="IE=edge">
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<title>Popup | Homework Kottans</title>
+	<link href="https://fonts.googleapis.com/css?family=Noto+Sans:regular,700&display=swap" rel="stylesheet">
+	<link rel="stylesheet" href="./css/nulling-styles.css">
+	<link rel="stylesheet" href="./css/styles.css">
+</head>
+<body>
+	<header class="header">
+		<div class="wrapper">
+			<div class="header__container">
+				<a href="#" class="header__logo">PopUp</a>
+				<div class="wrapper-popup">
+					<label for="popap-icon" class="label">
+						<svg version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" width="35" height="35" viewBox="0 0 16 16">
+							<path fill="#212121" d="M0 7h7v-7h-7v7zM1 1h5v5h-5v-5z"></path>
+							<path fill="#212121" d="M9 0v7h7v-7h-7zM15 6h-5v-5h5v5z"></path>
+							<path fill="#212121" d="M0 16h7v-7h-7v7zM1 10h5v5h-5v-5z"></path>
+							<path fill="#212121" d="M9 16h7v-7h-7v7zM10 10h5v5h-5v-5z"></path>
+						</svg>
+					</label>
+					<input type="checkbox" id="popap-icon" class="popup-button">
+					<div class="popup">
+						<nav>
+							<ul class="popup__list">
+								<li class="popup__item"><a class="popup__link" href="#"><img class="popup__img" src="./img/01.svg" alt="Спидометр">Спидометр</a></li>
```

Comment:
Read how and when to populate alt attribute value [Как правильно написать alt-текст](https://htmlacademy.ru/blog/articles/alt-text)

Reply:
Well, for that matter: https://skr.sh/sFU2dv2Z73p)))
But in fact, I am a SEO-specialist and I can write alt-s, I had quite a lot of practice. In general, it’s not developers who do this, but SEO-specialists. They make up templates, etc. Of course, I think it’s a good practice to fill them out (if it’s some kind of landing page, for example), but if the site is an online store?)

Reply:
So you think populating `alt` attribute for icons will be a good idea?

Reply:
I think that the alt should always be filled

Reply:
To avoid such situations, I prefer an approach in which icons are added as a background image.

Reply:
As far as I know, each page has its own context (the topic of the page) and each image should be in the context of the page.
For example: if the page of an online store with a certain product sets the main context for this product or its category, and when we have images (icons that we added via `img`) in which the content of the `alt` attribute differs from the context of the page, then this confuses the search bot that leads to a decrease in search rankings, and when the attribute is empty, the bot adds a description for itself or ignores this image altogether.

Reply:
Well, to each his own) I don't find it practical to add them in the background. Indeed, in case of need for replacement, there will be more trouble ... and so, the client can replace the image himself (I mean after integration into the CMS)

Reply:
I don’t quite understand what you mean, why should the alt attribute have a name out of context?
Well, in fact, alt in the picture does not greatly affect the ranking of the page in the search. It all depends on where the thematic "keys" are located. Keys affect the title tag the most, followed by h1, description, and so on. And the keys in alt are much less significant.

Alt rather affects the ranking of the image itself. And this is important, because there is a separate tab for images in the search.

I don’t know how clumsy this all sounds, Google translator after all ((

Reply:
Is there a need for improvement here?

Reply:
Your previous comment is valid for content images, but icons are decorative images
![image](https://user-images.githubusercontent.com/16196199/185793182-10d76507-f9a2-460b-872c-f01370bc787d.png)

Reply:
I meant in general all images. Participating in [image search](https://skr.sh/sFYJs1yPKkn) results is a good thing anyway. Therefore, the alt will not be superfluous. However, you decide here. ok, alt removed